research/sentiment_analysis/sentiment_main.py

```diff
@@ -0,0 +1,188 @@
+"""The model makes use of concatenation of two CNN layers with different kernel sizes.
+`sentiment_model.py` for more details about the models.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# pylint: disable=g-bad-import-order
+from absl import app as absl_app
+from absl import flags
+import tensorflow as tf
+# pylint: enable=g-bad-import-order
+
```

Comment:
nit: here we need to add the # pylint: disable=g-bad-import-order before our own module imports.